portal/common/lib/crypto.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+const { subtle } = globalThis.crypto;
```

Comment:
😂 I love that they called the crypto library `subtle`.

Reply:
Question: this works with node/server side js as well, right?

Reply:
Yes it works, here are the docks for nodejs: https://nodejs.org/api/webcrypto.html#cryptosubtle 

The server portal preview deployment is also working, so that's a proof too.